README.md

```diff
@@ -4,8 +4,54 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/Peripli/service-manager)](https://goreportcard.com/report/github.com/Peripli/service-manager)
 [![Coverage Status](https://coveralls.io/repos/github/Peripli/service-manager/badge.svg?branch=master)](https://coveralls.io/github/Peripli/service-manager?branch=master)
 
-Service Manager Core Module
+## What is Service Manager
 
-[Run Service Manager on CF](deployment/cf/README.md)
+Service Manager is a central registry for service brokers and platforms registration. It tracks service instances creation and allows sharing of service instances between different Platform Instances.
 
-[Run Service Manager on Kubernetes](deployment/k8s/README.md)
+## Setup Service Manager Components
+
```

Comment:
The overall setup of the solution consist a single installation of Service Manager component and one or more Service Broker Proxy components that runs on each one of the registered inside Platforms. 
In short, the Service Manager component is acting as the central repository and Service Broker proxies are the active components on the platform side. 
More details about Service Broker Proxy functionality can be found [here](https://github.com/Peripli/service-broker-proxy).